packages/gestalt/src/style.js

```diff
@@ -16,11 +16,13 @@ order in which you do so doesn't really matter.
 
 */
 
+type InlineStyle = { [key: string]: string | number };
+
 // TODO: This type should be opaque, however the Babel parser doesn't support
 //       the opaque syntax yet.
 type Style = {|
   className: Set<string>,
-  inlineStyle: { [key: string]: any },
```

Comment:
We can be more specific with `string | number` here because those are the only two types that work for a style key https://reactjs.org/docs/dom-elements.html#style